docs/guides/javascript.rst

```diff
@@ -339,6 +339,30 @@ Calls an Elgg action with the data passed. This handles outputting of system mes
       }
    });
 
+Module ``elgg/Spinner``
+~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Seems like sphinx doesn't like this header notation. It may work to switch to all dashes